<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextBrowser</class>
 <widget class="QMainWindow" name="TextBrowser">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>502</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Impact</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(240, 240, 240);
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
	border-radius: 5px;                          
	border: 15 px solid red;
	background-color :rgba(220, 220, 220, 200);
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditIP">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0,0, 0);
border-radius: 5px;                         
border: 1px solid #000000;
background-color :rgb(255, 255, 255);
</string>
         </property>
         <property name="placeholderText">
          <string>Enter IP</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="pushButtonSend">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>21</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0,0, 0);
border-radius: 5px;                         
border: 1px solid #000000;
background-color :rgb(255, 255, 255);
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::MailForward"/>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLineEdit" name="lineEditMessage">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0,0, 0);
border-radius: 5px;                         
border: 1px solid #000000;
background-color :rgb(255, 255, 255);
</string>
         </property>
         <property name="placeholderText">
          <string>Message</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEditName">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0,0, 0);
border-radius: 5px;                         
border: 1px solid #000000;
background-color :rgb(255, 255, 255);
</string>
         </property>
         <property name="placeholderText">
          <string>Enter Your Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="3">
        <widget class="QPushButton" name="pushButtonConnect">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0,0, 0);
border-radius: 5px;                         
border: 1px solid #000000;
background-color :rgb(255, 255, 255);
</string>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="5">
        <widget class="QTextBrowser" name="textBrowserMesseges">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0,0, 0);
border-radius: 5px;                         
border: 1px solid #000000;
background-color :rgb(255, 255, 255);
</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="emojiButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 0px;</string>
         </property>
         <property name="text">
          <string>ðŸ˜€</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>519</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
	border-radius: 5px;                          
	border: 15 px solid red;
	background-color :rgba(220, 220, 220, 200);
}</string>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuChats">
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Chats</string>
    </property>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuChats"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
